src/Servers/Kestrel/Core/src/IHttpsConfigurationHelper.cs

```diff
@@ -0,0 +1,22 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.AspNetCore.Server.Kestrel.Https.Internal;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core;
+
+internal interface IHttpsConfigurationHelper
+{
+    bool IsEnabled { get; }
+    void Enable(
+        IHostEnvironment hostEnvironment,
+        ILogger<KestrelServer> serverLogger,
+        ILogger<HttpsConnectionMiddleware> httpsLogger);
```

Comment:
Initialize instead of Enabled?

e.g. `IsInitialized` and `Initialize(...)`

Reply:
I'm fine either way.

Reply:
Done.